src/components/organisms/navbar.js

```diff
@@ -1,19 +1,41 @@
-import React from "react"
+import React, { useState } from "react"
 import PropTypes from "prop-types"
 import Navigation from "../molecules/navigation"
+import { Header } from "../../styles/header-styles"
+import blueRingImage from "../../images/logos/ring-blue.png"
 
-const NavBar = () => (
-  <header>
-    <Navigation />
-  </header>
-)
+const NavBar = ({ isHome }) => {
+  const [hamburgerState, setHamburgerState] = useState(false)
+
+  return (
+    <Header isSubpage={!isHome} mobileMenu={hamburgerState}>
+      <a className="site-identity" href="/">
+        <h1>In the fight</h1>
+        <img src={blueRingImage} alt="blue ring" />
+      </a>
+      <div className="nav-wrapper">
+        <Navigation />
+      </div>
+
+      <button
+        id="hamburger-menu"
+        onClick={() => setHamburgerState(prevState => !prevState)}
+      >
+        <div></div>
+        <div></div>
+      </button>
+    </Header>
+  )
+}
 
 NavBar.propTypes = {
   siteTitle: PropTypes.string,
+  isHome: PropTypes.Bool,
```

Comment:
There are a couple issues with the build on this Proptype, looks like it just needed to be lowercase!
```suggestion
  isHome: PropTypes.bool,
```